src/lib/y2users/linux/useradd_config_writer.rb

```diff
@@ -0,0 +1,174 @@
+# Copyright (c) [2021] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "yast/i18n"
+require "yast2/execute"
+
+module Y2Users
+  module Linux
+    # Configures useradd in the target system according to a given {UseraddConfig} object
+    class UseraddConfigWriter
+      include Yast::I18n
+      include Yast::Logger
+      Yast.import "ShadowConfig"
+
+      # Constructor
+      #
+      # @param config [Y2User::Config] see #config
```

Comment:
Wrong namespace in documentation, missing `s`. Same happens several times,  please check.

```suggestion
      # @param config [Y2Users::Config] see #config
```

Reply:
This was copied&pasted from `Linux::Writer` which, apart from being wrong, redundantly specifies the namespace everywhere. This proves that's not a great idea, so I will remove the excess of namespaces in both files.

Reply:
Fixed now.